rules/recognizing-phishing-urls/rule.md

```diff
@@ -26,40 +26,46 @@ For the address <https://www.ssw.com.au/services>, services is the path. The pat
 
 You should always check that the **domain** matches the service or website you are expecting.
 
+::: greybox
 <http://linkedin.com.sggr.ru/someaddress>
+:::
+::: bad
+Bad Example – the address has LinkedIn in it, but it is a sub-domain, not the domain
+:::
 
-❌Bad example – the address has LinkedIn in it, but it is a sub-domain, not the domain
-
- 
 
+::: greybox
 <http://linked-in-hq.com/linkedin/myprofile>
-
-❌Bad example – the address has LinkedIn in it, but it is in the path, not the FQDN. The FQDN is also suspicious.
+:::
+::: bad
+Bad Example – the address has LinkedIn in it, but it is in the path, not the FQDN. The FQDN is also suspicious.
+:::
 
  
-
+::: greybox
 [http://linkedinalerter.com](http://anzalertercom/)
-
-❌Bad example – the address has LinkedIn in it, but is not a legitimate LinkedIn site.
-
- 
+:::
+::: bad
+Bad Example – the address has LinkedIn in it, but is not a legitimate LinkedIn site.
```

Comment:
Tiny one - Please remove full stop from this and other captions (for consistency)